metricbeat/module/system/raid/raid_test.go

```diff
@@ -34,6 +78,7 @@ func TestData(t *testing.T) {
 }
 
 func TestFetch(t *testing.T) {
+
```

Comment:
I have noticed that you use to add an empty line after function definitions. I think we don't have a convention regarding that, and I don't have an strong opinion, but I'd suggest to remove them to be more consistent with the existing code.

Reply:
Just make sure @andrewkroh does not see it ;-)

Reply:
Yah, I guess I do do that, don't I?

Reply:
😆 

You can police your code with my tool: https://github.com/andrewkroh/go-nonewlines#example

Reply:
Haha, great tool :+1: